src/vm/ngenhash.inl

```diff
@@ -151,7 +151,7 @@ void NgenHashTable<NGEN_HASH_ARGS>::BaseInsertEntry(NgenHashValue iHash, VALUE *
     pVolatileEntry->m_iHashValue = iHash;
 
     // Compute which bucket the entry belongs in based on the hash.
-    DWORD dwBucket = iHash % m_cWarmBuckets;
+    DWORD dwBucket = iHash % VolatileLoad(&m_cWarmBuckets);
```

Comment:
Nit: It should not be needed here since this runs under a lock

Reply:
Removed the volatile read